homeassistant/components/camera/synology.py

```diff
@@ -212,6 +221,14 @@ def async_read_sid(self):
             self._valid_cert
         )
 
+    def websession(self):
+        """Create a Client Session."""
+        if self._websession is None:

```

Comment:
Except I need a different connector to pass the cookie.  aiohttp.get accepts cookies, but aiohttp.clientsession.get does not take a cookie parameter, only the connector does.